### DISCUSSION TITLE
Block width

### DISCUSSION THREAD

**Discussion by ArianeNeves** [ORTools4MTEX/ORTools#15] [Q&A] [answered]
Hi Frank,
I am analyzing a martensitic stainless steel EBSD map. I'm trying to calculate the block width. The blocks are irregular. 

I followed your tutorial (example 7) and performed the average block width calculation for the martensite data without errors.

However, when I try to perform the calculation for my sample, Mtex gives the following error: 
"Undefined function or variable 'calcBlockWidth'".

May you can help me?

Thank you very much!

**Comment by frankNiessen** [comment, marked as answer]
Hi Ariane, 

can you confirm that you are running the latest version [ORTools v.2.0](https://github.com/ORTools4MTEX/ORTools/releases/tag/2.0.0) ?

best wishes
Frank

**Comment by ArianeNeves** [comment]
Frank, thanks for the help. I was not running the latest version.
I found the 'calcBlockWidth' function.
